recipe-server/normandy/recipes/api/serializers.py

```diff
@@ -96,6 +97,20 @@ def create(self, validated_data):
 
         return self.update(recipe, validated_data)
 
+    def validate_extra_filter_expression(self, value):
+        jexl = JEXL()
+
+        # Add mock transforms for validation
+        jexl.add_transform('date', lambda x: x)
+        jexl.add_transform('stableSample', lambda x: x)
+        jexl.add_transform('bucketSample', lambda x: x)
```

Comment:
This isn't very satisfying, but I don't know how we can do better. Maybe we can put a link to the docs that say what transforms need to be available?